src/api/status/public/js/build-log/build-header.js

```diff
@@ -0,0 +1,38 @@
+const buildHeaderTitle = document.getElementById('build-header-title');
+const buildHeaderInfo = document.getElementById('build-header-info');
+const buildSender = document.getElementById('build-sender');
+const buildSenderName = document.getElementById('build-sender-name');
+const buildSenderImg = document.getElementById('build-sender-img');
+const buildGitSHA = document.getElementById('build-git-sha');
+const buildResult = document.getElementById('build-result');
+const buildStarted = document.getElementById('build-started');
+const buildDuration = document.getElementById('build-duration');
+
+function renderBuildInfo({ githubData, startedAt, stoppedAt, result }) {
+  buildHeaderTitle.innerHTML = '';
```

Comment:
Line 12 has changed, but it is different from Andrent219's comment.
![image](https://user-images.githubusercontent.com/55161917/145731065-ef97deba-e9e5-4434-9fd8-d7ecb5cde486.png)